types/diff-match-patch/index.d.ts

```diff
@@ -1,22 +1,19 @@
-// Type definitions for diff-match-patch v1.0.0
+// Type definitions for diff-match-patch 1.0
 // Project: https://www.npmjs.com/package/diff-match-patch
-// Definitions by: Asana <https://asana.com>
+// Definitions by: Asana <https://github.com/asana>
```

Comment:
Maintainer requested that you leave this as `https://asana.com` - you might need to add `"rules": { "dt-header": false }` to the `tslint.json`.

Reply:
OK, if it's allowed.

Reply:
Yeah, we just like to avoid disabling it where possible.